bundles/org.openhab.binding.liquidcheck/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="liquidcheck"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="liquidCheckDevice">
+		<label>Liquid Check Device</label>
+		<category>Sensor</category>
+
+		<channels>
+			<channel id="content" typeId="content-channel"/>
+			<channel id="level" typeId="level-channel"/>
+			<channel id="fill-indicator" typeId="fill-indicator-channel"/>
```

Comment:
I'm really sorry. There are currently some ongoing discussions about the naming convention which includes also channel ids. Therefore it's up to you if you would like to use the suggested **camelCase** or if you prefer **with-hyphens** as you originally used.

System channel types use the latter, and the convention might adopt this, although I can't say for sure yet. Although it will admittedly be annoying to change this once again, there won't be a better time than now, since it will be a breaking change after the binding has been released. But as said, it will be up to you.

Again, I'm sorry.

Reply:
It's up to you, since a naming convention is not settled yet. My personal recommendation would probably now be `lowercase-with-hyphens` as I have seen best arguments for that convention.

Reply:
Please remember the README also.

Reply:
... yeah, have overseen that again ...